autopilot/contractor.go

```diff
@@ -141,6 +141,8 @@ func (c *contractor) performContractMaintenance(ctx context.Context, cfg api.Aut
 		if err != nil {
 			return fmt.Errorf("failed to determine min score for contract check: %w", err)
 		}
+	} else {
+		c.logger.Warn("could not calculate min score, no hosts found")
```

Comment:
(nit) oh wait we got this line twice now. Once here and once in `managedFindMinAllowedHostScores`. Probably fine to keep the latter but update it to not say `0` anymore.

Reply:
updated